types/index.ts

```diff
@@ -9,3 +9,22 @@ export interface CustomButtonProps {
   rightIcon?: string;
   isDisabled?: boolean;
 }
+
+export interface ChapterCardProps {
+  title: string;
+  subtitle: string;
+  image: string;
+  exercises: string;
+  link: string;
+}
+
+export interface Info {
+  id: number;
+  title: string;
+  details: string;
+  code(name: string): void;
+}
+
+export interface PageCardProps {
+  cardInfo: Info[];
+}
```

Comment:
Can we move each type to the same file of each component?

Reply:
Refactored every component including its type